javascript/javascript3/week1/own-json.js

```diff
@@ -1,5 +1,5 @@
 //Create your own json file of interest
-// valid
+// Json file valid
 const myDetail = `{
```

Comment:
Don't use ' other wise it will be String not the object. and JSON should return object.